ArduPlane/servos.cpp

```diff
@@ -685,10 +685,23 @@ void Plane::set_servos_flaps(void)
         manual_flap_percent = channel_flap->percent_input();
     }
 
-    if (control_mode->does_auto_throttle()) {
+    const auto flap_actual_speed = flight_option_enabled(FlightOptions::FLAP_ACTUAL_SPEED);
+    const bool has_target_airspeed = control_mode->does_auto_throttle();
+    if (has_target_airspeed || flap_actual_speed) {
         int16_t flapSpeedSource = 0;
-        if (ahrs.using_airspeed_sensor()) {
+        float est_airspeed;
+        bool have_airspeed = ahrs.airspeed_estimate(est_airspeed);
```

Comment:
```suggestion
        const bool have_airspeed = ahrs.airspeed_estimate(est_airspeed);
```

or could be moved into the one place it is used and save the call when we have a target and we're using a asensor